### DISCUSSION TITLE
Kerosene Start

### DISCUSSION THREAD

**Discussion by Marian-K** [Jehanzeb1973/Ardu_ECU#12] [Ideas]
-Can You add the kerosene Start function?

- for two Stage Turbines a second optional Rpm Sensor Witz Control parameters for a constant speed on the second shaft

**Comment by Jehanzeb1973** [comment]
Hi MarianKetostart version is already developed and is successfully tested. I am just waiting to collect more data to ensure no major bugs before release. Conceptually dual shaft engine control is not difficult. The startup is based on main shaft RPM and operating can be logically linked to RPM from second spool. I won't be able to test it as I don't have such an engineRegards Jehanzeb Sent from my phone-------- Original message --------From: Marian-K ***@***.***>Date: Sun, Feb 11, 2024, 9:55 PMTo: Jehanzeb1973/Ardu_ECU ***@***.***>Cc: Subscribed ***@***.***>Subject: [Jehanzeb1973/Ardu_ECU] Kerosene Start (Discussion #12)
-Can You add the kerosene Start function?
for two Stage Turbines a second optional Rpm Sensor Witz Control parameters for a constant speed on the second shaft
